urllib3/connectionpool.py

```diff
@@ -204,8 +204,9 @@ def _new_conn(self):
         Return a fresh :class:`HTTPConnection`.
         """
         self.num_connections += 1
-        log.debug("Starting new HTTP connection (%d): %s",
-                  self.num_connections, self.host)
+        log.debug("Starting new HTTP connection (%d): %s:%s",
+                  self.num_connections, self.host,
+                  self.port if self.port else "80")
```

Comment:
Nit: here and below, you can use the idiom `self.port or "80"` to obtain the first truthy value.